apps/wing/src/cli.ts

```diff
@@ -227,6 +227,12 @@ async function main() {
         .preset(3)
         .argParser(parseInt)
     )
+    .addOption(
+      new Option(
+        "-b, --batch [batch]",
+        "Number of tests to be executed on parallel- if not specified will execute all at the same time"
```

Comment:
This description and the PR description don't quite match the implementation.

If you have 10 tests with `-b 2`. If the first test takes 2 minutes and rest take 1 second, you'll waste a lot of time on the first batch. Ideally, the first one would run while the rest run 1-at-a-time in parallel with the first.

If possible, I would rename this option to `parallelism` and maybe use https://www.npmjs.com/package/p-limit (or a custom implementation if it's simple enough)